src/css/_c4aa-clipPaths.css

```diff
@@ -61,9 +61,9 @@ header:not(.featured-image) h1.entry-title,
 		0 0, 
 		100% 0, 
 		100% 
-		calc( 100% - ( var( --bottomRightY ) * 2px ) ),
+		calc( 100% - ( var( --bottomRightY ) * 3px ) ),
 		0 
-		calc( 100% - ( var( --bottomLeftY ) * 2px ) )
+		0 /* calc( 100% - ( var( --bottomLeftY ) * 3px ) )*/
```

Comment:
Can you add a message to the commented code - update it to: "Made more extreme angles, keeping this here in case we want to revert - calc( 100% - ( var( --bottomLeftY ) * 3px ) )"

Reply:
I took it out because we have it recorded here.